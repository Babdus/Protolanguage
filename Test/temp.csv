, en, fr, de, ru
house, house, maison, haus, дом
dog, dog, , hund,
cat, , chat, , кошка
