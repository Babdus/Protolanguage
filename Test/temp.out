3 egy
8 képesség
5 képes
6 ról ről
6 felett
7 elfogad
7 szerint
6 számla
5 át
7 törvény
6 akció
10 tevékenység
12 tulajdonképpen
7 hozzáad
8 cím
5 adminisztráció
7 beismerni
5 felnőtt
5 érint
6 után
6 újra
5 ellen
5 kor
7 ügynökség
5 ügynök
5 ezelőtt
8 egyetért
10 megegyezés
5 előre
5 levegő
7 minden
13 lehetővé teszi
6 majdnem
8 egyedül
6 mentén
7 már
7 is
5 habár
5 mindig
2 Amerikai
6 között
8 összeg
8 elemzés
3 és
5 állat
8 egy másik
7 válasz
8 bármilyen
6 bárki
5 bármi
8 megjelenik
6 alkalmaz
12 megközelítés
8 terület
8 érvel
2 kar
7 körül
9 megérkezik
7 Művészet
6 cikk
9 művész
3 mint
5 kérdez
12 feltételezni
6 nál nél
6 támadás
7 Figyelem
7 ügyvéd
8 közönség
7 szerző
9 hatóság
9 elérhető
7 elkerül
5 el
3 baba
6 hát
6 rossz
5 táska
4 labda
1 bank
1 bár
4 bázis
4 lenni
3 üt
6 szép
5 mert
7 válik
4 ágy
7 előtt
7 kezdődik
9 viselkedés
6 mögött
5 hinni
7 haszon
7 legjobb
4 jobb
6 között
7 túl
3 nagy
6 számla
15 milliárd, ezermillió
4 bit
5 fekete
4 vér
6 kék
2 tábla
4 test
6 könyv
9 született
7 mindkét
3 doboz
4 fiú
6 szünet
6 hoz
11 fiú testvér
11 költségvetés
10 épít
7 épület
6 üzleti
5 de
5 megvesz
7 által
7 hívás
0 kamera
4 kampány
4 tud
8 rák
8 jelölt
8 főváros
2 autó
5 kártya
11 gondoskodás
2 karrier
6 visz
4 ügy
11 fogás
5 ok
5 sejt
6 központ
8 központi
7 század
7 bizonyos
10 biztosan
4 szék
7 kihívás
7 véletlen
10 változás
9 karakter
9 díj
9 jelölje be
5 gyermek
8 választás
7 választ
6 templom
10 állampolgár
5 város
2 civil
12 követelés
7 osztály
9 egyértelmű
12 tisztán
7 Bezárás
14 távolsági busz
4 hideg
10 Gyűjtemény
12 főiskola
6 szín
6 jön
9 kereskedelmi
6 közös
10 közösség
8 vállalat
13 hasonlítsa össze
11 számítógép
8 vonatkozik
7 feltétel
10 konferencia
3 Kongresszus
5 fontolgat
12 fogyasztó
9 tartalmaz
8 Folytatni
8 ellenőrzés
5 költség
6 tudott
6 ország
5 párosít
8 tanfolyam
9 bíróság
6 borító
6 teremt
6 bűn
3 kulturális
1 kultúra
6 csésze
6 jelenlegi
5 vevő
6 vágott
5 sötét
4 adat
4 lánya
4 nap
5 halott
6 üzlet
7 halál
7 vita
8 évtized
11 döntsd el
7 döntés
6 mély
7 védelem
7 fokozat
1 Demokrata
3 demokratikus
10 leírni
8 tervezés
11 annak ellenére
9 Részlet
11 meghatározására
8 fejleszt
10 fejlesztés
7 meghal
8 különbség
10 különböző
5 nehéz
5 vacsora
8 irány
7 rendező
9 felfedez
10 megvitatása
7 vita
11 betegség
6 csinál
7 orvos
6 kutya
2 ajtó
5 le-
9 húz
6 álom
6 hajtás
5 csepp
1 drog
5 alatt
11 minden egyes
4 korai
6 keleti
7 könnyen
5 eszik
10 gazdasági
8 gazdaság
4 él
7 oktatás
6 hatás
9 erőfeszítés
6 nyolc
7 bármelyik
10 választás
6 más
9 munkavállaló
3 vég
0 energia
8 élvez
7 elég
7 belép
6 teljes
9 környezet
9 környezeti
9 különösen
8 létrehozni
3 még
10 este
7 esemény
6 valaha
7 minden
5 mindenki
5 mindenki
4 minden
8 bizonyíték
8 pontosan
7 példa
10 végrehajtó
8 létezik
4 elvár
10 tapasztalat
7 szakértő
11 megmagyarázni
4 szem
6 arc
4 tény
7 tényező
10 nem sikerül
4 esik
6 család
6 messze
6 gyors
2 apa
7 félelem
9 szövetségi
8 érez
10 érzés
5 kevés
4 mező
6 harc
5 ábra
4 tölt
0 film
5 végső
7 végül
7 pénzügyi
10 megtalálja
6 bírság
4 ujj
7 Befejez
4 Tűz
4 cég
8 első
5 hal
3 öt
6 padló
4 légy
3 fókusz
8 kövesse
9 élelmiszer
5 láb
6 mert
8 erő
9 külföldi
8 elfelejt
6 forma
6 korábbi
5 előre
4 négy
7 ingyenes
4 barát
6 tól től
7 elülső
6 teljes
6 alap
9 jövő
14 játszma, meccs
7 kert
2 gáz
7 Tábornok
9 generáció
5 kap
5 lány
4 adni
5 üveg
4 megy
7 cél
3 jó
5 kormány
5 nagy
5 zöld
7 talaj
7 csoport
6 nő
8 növekedés
8 Találd ki
2 pisztoly
10 fickó
4 haj
5 fél
4 kéz
4 lóg
6 történik
7 boldog
6 kemény
4 van
2 ő
5 fej
6 Egészség
6 hall
6 szív
5 hőség
4 nehéz
7 Segítség
3 neki
5 itt
9 önmaga
6 magas
4 neki
9 saját maga
5 övé
9 történelem
7 találat
5 tart
7 itthon
6 remény
8 kórház
4 forró
7 szálloda
3 óra
4 ház
4 hogyan
6 azonban
7 hatalmas
6 emberi
4 száz
6 férj
2 én
5 ötlet
13 azonosítani
3 ha
6 kép
10 Képzeld el
9 hatás
10 fontos
6 javul
6 ban ben
12 tartalmaz
6 beleértve
9 növekedés
12 valóban
6 jelez
8 Egyedi
7 ipar
3 információ
7 belül
7 helyette
9 intézmény
8 érdeklődés
10 érdekes
10 nemzetközi
11 interjú
5 -ba
8 beruházás
5 vonja
6 probléma
3 azt
8 tétel
5 annak
3 maga
3 munka
9 csatlakozik
7 éppen
4 tart
5 kulcs
8 kölyök
8 megöl
6 kedves
6 konyha
8 tud
7 tudás
6 föld
9 nyelv
6 nagy
6 utolsó
7 késő
11 majd később
7 nevetés
7 törvény
6 jogász
6 világi
4 vezet
5 vezető
6 tanul
10 legkevésbé
9 szabadság
5 balra
5 láb
3 jogi
8 Kevésbé
5 enged
5 levél
4 szint
6 hazugság
6 élet
5 fény
8 mint
11 valószínűleg
4 vonal
7 lista
7 hallgat
5 kicsit
4 élő
5 helyi
5 hosszú
8 néz
7 elveszít
8 veszteség
3 sok
6 szeretet
8 alacsony
6 gép
9 magazin
7 fő-
7 fenntart
10 Jelentősebb
7 többség
6 csinál
5 Férfi
6 kezel
7 vezetés
9 menedzser
3 sok
7 piac
10 házasság
6 anyag
7 ügy
5 lehet
4 talán
4 nekem
9 átlagos
9 intézkedés
1 média
6 orvosi
8 találkozik
8 találkozó
5 tag
1 memória
6 említés
5 üzenet
7 eljárás
7 középső
7 esetleg
8 katonai
3 millió
6 elme
6 perc
7 hiányzik
8 küldetés
5 modell
2 modern
7 pillanat
5 pénz
6 hónap
7 több
6 reggel
8 a legtöbb
2 anya
4 száj
8 mozog
8 mozgalom
5 film
5 úr
7 Asszony
5 sokkal
6 zene
7 kell
5 az én
7 magamat
4 név
7 nemzet
9 nemzeti
11 természetes
9 természet
5 közel
5 közel
11 szükséges
7 szükség
6 hálózat
7 soha
9 új
4 hírek
8 újság
9 következő
6 szép
6 éjszaka
2 nem
9 egyik sem
4 sem
10 északi
2 nem
7 jegyzet
4 semmi
9 értesítés
5 Most
2 nem
8 szám
8 előfordul
8 nak,-nek
3 ki
8 ajánlat
7 hivatal
6 tiszt
7 hivatalos
5 gyakran
2 ó
3 olaj
6 rendben
7 régi
7 tovább
7 egyszer
3 egy
6 csak
4 -ra
8 nyisd ki
9 művelet
8 lehetőség
18 választási lehetőség
4 vagy
3 sorrend
11 szervezet
4 más
10 mások
4 mi
3 ki
4 kívül
6 felett
7 saját
9 tulajdonos
4 oldal
7 fájdalom
11 festés
1 papír
6 szülő
5 rész
13 résztvevő
8 különös
9 különösen
6 partner
4 party
7 elhalad
7 múlt
9 türelmes
5 minta
8 fizetés
2 béke
6 emberek
0 per
7 teljesít
9 teljesítmény
4 talán
6 időszak
8 személy
10 személyes
8 telefon
3 fizikai
4 szed
5 kép
5 darab
3 hely
4 terv
7 növény
6 játék
7 játékos
11 DÉLUTÁN
2 pont
10 rendőrség
8 irányelv
2 politikai
0 politika
7 szegény
8 népszerű
8 népesség
4 pozíció
5 pozitív
9 lehetséges
4 erő
6 gyakorlat
6 készít
7 ajándék
8 elnök
6 nyomás
6 szép
10 megelőzése
6 ár
8 magán
11 valószínűleg
6 probléma
7 folyamat
8 gyárt
7 termék
9 Termelés
11 szakmai
13 Egyetemi tanár
1 program
5 program
6 ingatlan
7 véd
7 bizonyít
9 biztosítani
7 nyilvános
5 Húzni
6 célja
8 nyom
5 tesz
8 minőség
6 kérdés
6 gyorsan
7 egészen
7 verseny
2 rádió
8 emel
11 hatótávolság
5 arány
6 Inkább
8 elérés
8 olvas
4 kész
7 igazi
9 valóság
12 megvalósítani
6 igazán
7 ok
3 kap
10 friss
10 mostanában
7 elismerik
7 rekord
5 piros
12 csökkentésére
8 tükrözik
8 vidék
8 viszonyul
8 kapcsolat
6 vallási
6 marad
9 emlékezik
9 eltávolít
7 jelentés
10 képvisel
14 Köztársasági
6 kíván
7 kutatás
9 forrás
9 reagál
9 válasz
12 felelősség
7 pihenés
7 eredmény
11 Visszatérés
6 felfed
7 gazdag
9 jobb
8 emelkedik
8 kockázat
8 út
5 szikla
5 szerep
4 szoba
6 szabály
12 fuss
10 biztonságos
6 azonos
6 mentés
4 mond
6 színhely
1 iskola
8 tudomány
11 tudós
5 pontszám
6 tenger
9 évszak
8 ülés
8 második
5 szakasz
9 Biztonság
5 lát
9 törekszik
8 látszik
4 elad
5 elküld
7 idősebb
6 érzék
5 sorozat
6 komoly
12 szolgál
10 szolgáltatás
7 készlet
5 hét
7 számos
3 szex
4 szexuális
7 ráz
7 részvény
4 ő
4 lő
8 rövid
10 lövés
10 kellene
7 váll
10 előadás
5 oldal
4 jel
11 jelentős
7 hasonló
6 egyszerű
11 egyszerűen
8 mivel
5 énekel
8 egyetlen
10 lánytestvér
5 ül
6 webhely
6 helyzet
3 hat
6 méret
8 jártasság
5 bőr
4 kicsi
9 mosoly
6 így
8 társadalmi
9 társadalom
7 katona
6 néhány
6 valaki
6 valaki
6 valami
8 néha
3 fiú
6 dal
4 hamar
6 fajta
5 hang
5 forrás
4 déli
9 déli
8 tér
8 beszél
10 különleges
11 különleges
7 beszéd
8 költ
5 Sport
6 tavaszi
9 személyzet
5 színpad
10 állvány
14 alapértelmezett
5 csillag
6 Rajt
6 állapot
10 nyilatkozat
6 állomás
10 tartózkodás
5 lépés
6 még mindig
6 Készlet
7 állj meg
5 bolt
4 sztori
2 stratégia
5 utca
6 erős
8 szerkezet
7 diák
9 tanulmány
5 dolog
3 stílus
7 tantárgy
8 siker
8 sikeres
13 ilyen
8 hirtelen
7 szenvedni
10 javasol
5 nyári
8 támogatás
5 biztos
6 felület
8 rendszer
5 asztal
5 vesz
9 beszélgetés
6 feladat
5 adó
8 tanít
7 tanár
6 csapat
3 technológia
6 televízió
5 mond
5 tíz
8 hajlamosak
7 kifejezés
5 teszt
3 mint
8 köszönet
2 hogy
2 a
5 azok
7 őket
5 maguk
6 azután
7 elmélet
3 ott
4 ezek
4 ők
5 dolog
6 gondol
7 harmadik
3 ez
5 azok
4 bár
6 gondolat
4 ezer
9 fenyegetés
3 három
7 keresztül
5 egész
4 dobás
6 és így
7 idő
7 nak nek
3 Ma
9 együtt
8 ma este
6 is
5 felső
7 teljes
7 kemény
5 felé
4 város
10 kereskedelmi
11 hagyományos
11 kiképzés
6 utazás
9 csemege
10 kezelés
6 fa
6 próba
6 utazás
6 baj
4 IGAZ
6 igazság
10 próbáld ki
7 fordulat
4 tévé
3 két
5 típus
7 alatt
5 megért
7 egység
3 amíg
4 fel
5 esetén
6 minket
8 használat
6 általában
5 érték
8 különféle
5 nagyon
7 áldozat
8 Kilátás
9 erőszak
8 látogatás
5 hang
8 szavazás
5 várjon
5 séta
5 fal
4 akar
5 háború
5 néz
3 víz
4 út
4 mi
7 fegyver
7 viselet
3 hét
5 súly
4 jól
10 nyugat
10 nyugati
3 mit
12 tök mindegy
5 amikor
2 hol
4 vajon
5 melyik
9 míg
5 fehér
3 ki
5 egész
4 kit
3 akinek
7 miért
5 széles
7 feleség
5 akarat
8 győzelem
5 szél
5 ablak
9 szeretnék
6 val vel
6 belül
6 nélkül
7 nő
8 csoda
4 szó
4 munka
7 munkás
5 világ
8 aggodalom
7 lenne
6 ír
8 író
4 rossz
6 udvar
4 igen
4 év
4 Igen
8 még
3 Ön
5 fiatal
3 a te
11 saját magad
